indexFinal.html

```diff
@@ -5,53 +5,49 @@
-                        <label> <input type="radio" name="Anrede" value="Frau"> Frau </label>
-                        <label> <input type="radio" name="Anrede" value="Herr"> Herr </label>
-                        <label> <input type="radio" name="Anrede" value="Keine-Angabe"> Keine Angabe </label>
-                    </fieldset>
-            
-                    <label for="name">Ihr Name</label>
-                    <input type="text" id="name" name="name"/>
-                    <div class="required-field"><span class="asterisk"> * </span> Required field </div>
-     
-                    <label for="email"> Ihr E-Mail-Adresse </label>
-                    <input type="email" id="email" name="email"/>
-                    <div class="required-field"><span class="asterisk"> * </span> Required field </div>
+                    <div class="left">
+                        
+                        <header>
+                            <h1>Schreiben Sie <br /> uns eine <br /> Nachricht</h1>
+                            <hr />
+                        </header>
 
-                    <div class="send-button-wrapper">
-                        <button type="submit" name="send-button" class="send-button" onclick= "inputValue()"> Send </button>
+                        <label for="message"> Ich interessiere mich für ...</label>
+                        <textarea id="message"></textarea>     
+                        <div class="required-field"><span class="asterisk"> * </span> Required field </div>
                     </div>
-
-                    <script src="scriptFinal.js"></script>
-
-                </div>
-
-            </form> 
+                    <div class="vertical"></div>
+                    <div class="right">
+                        <fieldset>
+                            <legend> Anrede </legend>
+                            <label> <input type="radio" name="Anrede" value="Frau"> Frau </label>
+                            <label> <input type="radio" name="Anrede" value="Herr"> Herr </label>
+                            <label> <input type="radio" name="Anrede" value="Keine-Angabe"> Keine Angabe </label>
+                        </fieldset>
+                    
+                        <label for="name">Ihr Name</label>
+                        <input type="text" id="name" name="name"/>
+                        <div class="required-field"><span class="asterisk"> * </span> Required field </div>
+                    
+                        <label for="email"> Ihr E-Mail-Adresse </label>
+                        <input type="email" id="email" name="email"/>
+                        <div class="required-field"><span class="asterisk"> * </span> Required field </div>
+                        <div class="send-button-wrapper">
+                            <button type="submit" name="send-button" class="send-button"> Send </button>
+                        </div>
+                        <script src="scriptFinal1.js"></script>
```

Comment:
Script should go at the bottom after the html closing tag.
This is for faster loading of the page, so the html can render first, and after that .js file is loaded